src/EntryPoints/Rest/GetAllMappingsApi.php

```diff
@@ -29,9 +29,9 @@ public function run(): array {
 				'text.old_id',
 			] )
 			->from( 'text' )
-			->join( 'page', conds: 'text.old_id=page.page_latest' )
 			->join( 'slots', conds: 'text.old_id=slots.slot_content_id' )
 			->join( 'slot_roles', conds: 'slots.slot_role_id=slot_roles.role_id' )
+			->join( 'page', conds: 'slots.slot_revision_id=page.page_latest' )
```

Comment:
This minor change links the page's latest revision:
![Screenshot_20220727_001801](https://user-images.githubusercontent.com/1428594/181121946-4ca20f5f-583f-437f-9908-cda0be0f9e28.png)

To `slots.slot_revision_id`:
![Screenshot_20220727_001733](https://user-images.githubusercontent.com/1428594/181121907-306ce2a4-2ae4-4eba-ac41-7ce16773c057.png)

From which we can get the 'slot_content_id` with `slot_role_id=2` (wikibase-rdf) = 30

Which gives us the right `text` row:
![Screenshot_20220727_001947](https://user-images.githubusercontent.com/1428594/181122139-005a3b27-0f3f-49e9-a105-fb433e676002.png)

Reply:
With this I get all the mappings in the API even when making non-mapping changes to an entity.

The comparable SQL query:
```sql
SELECT text.*, slots.*, page.*
FROM text
JOIN slots ON slots.slot_content_id=text.old_id
JOIN slot_roles ON slot_roles.role_id=slots.slot_role_id
JOIN page ON page.page_latest=slots.slot_revision_id
WHERE slot_roles.role_name = 'wikibase-rdf'
```

Reply:
And this obviously only gets the entites where a mapping slot value exists. If we need to return all entities to show which ones don't have mappings then this query won't work.